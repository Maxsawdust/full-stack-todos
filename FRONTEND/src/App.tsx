import { Routes, Route } from "react-router";
import { _Layout, Landing, Login, Signup, Dashboard } from "./pages";

function App() {
  return (
    <div className="h-screen flex bg-gray-100">
      <Routes>
        <Route path="/" element={<_Layout />}>
          <Route index element={<Landing />} />
          <Route path="/login" element={<Login />} />
          <Route path="/signup" element={<Signup />} />
          <Route path="/dashboard" element={<Dashboard />} />
        </Route>
      </Routes>
    </div>
  );
}

export default App;
