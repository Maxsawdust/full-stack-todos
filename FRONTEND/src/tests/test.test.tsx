import { MemoryRouter } from "react-router";
import App from "../App";
import { render, screen } from "@testing-library/react";

// test to make sure the testing suite works
describe("tests", () => {
  it("should render component", () => {
    render(
      <MemoryRouter>
        <App />
      </MemoryRouter>
    );
    expect(<App />).toMatchSnapshot();
    screen.debug();
  });
});
