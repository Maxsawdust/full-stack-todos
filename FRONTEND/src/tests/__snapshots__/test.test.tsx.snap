// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`tests > should render component 1`] = `<App />`;
